README.md

```diff
@@ -128,6 +128,9 @@ To reload the config, you'll need to run `swaync-client --reload-config`
 - `control-center-margin-right`: uint (Any positive number, 0 to disable). The margin (in pixels) at the right of the notification center
 - `control-center-margin-left`: uint (Any positive number, 0 to disable). The margin (in pixels) at the left of the notification center
 - `script-fail-notify`: bool. Sends a notification with error message if a script fails
+- `fit-to-screen`: `true` or `false`. If the control center should expand to fill the screen
+- `control-center-height`: uint (Any positive number). The control center's height in pixels, this is ignored when the `fit-to-screen` option is set to `true`
+- `control-center-width`: uint (Any positive number). The control center's width in pixels
```

Comment:
Just added a Man file for the configuration file. Make sure to remove this and add the config options there :)

Reply:
Done!